src/cart/line-items.mock.ts

```diff
@@ -29,6 +29,7 @@ export function getPhysicalItem(): PhysicalItem {
                 valueId: 3,
             },
         ],
+        categories: [[{name: 'Cat 1'}], [{name: 'Cat 2'}], [{name: 'Cat 3'}]],
```

Comment:
I think it might be nice if you can provide multiple category names for a category. i.e.: `[{ name: 'Furniture' }, { name: 'Bed' }]`. That way, you can better test your `map` function.